administrative-sdk/speechChallenge.js

```diff
@@ -177,24 +115,22 @@ class SpeechChallenge {
    * @param {Sdk~listSpeechChallengesCallback} cb The callback that handles the response.
    * @param {Sdk~listSpeechChallengesErrorCallback} [ecb] The callback that handles the error response.
```

Comment:
Missing documentation update. Callbacks are no longer taken and a promise is returned.